packages/toolbox/src/chainlinkConversionPathTools.ts

```diff
@@ -56,37 +54,35 @@ class ChainlinkConversionPathTools {
 
     this.creationBlockNumber = chainlinkConversionPath.getCreationBlockNumber(this.network);
 
-    this.maxRange = options?.maxRange || 5000;
+    this.maxRange = options?.maxRange || 1000000;
   }
 
   /**
    * Retrieves all the aggregators
    */
   public async getAggregators(): Promise<Record<string, Record<string, string>>> {
     const lastBlock = await this.provider.getBlockNumber();
-    const chunks = chunk(
-      Array(lastBlock - this.creationBlockNumber)
-        .fill(0)
-        .map((_, i) => this.creationBlockNumber + i),
-      this.maxRange,
-    );
```

Comment:
This piece of code is problematic and results in an out-of-memory error on networks with many blocks like Arbitrum. I replaced it with a simple while loop.